src/components/KeyblockMicroblocksPanel.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <app-panel
+    v-if="microblocks"
+    class="keyblock-microblock-panel">
+    <paginated-content
+      v-model:page-index="pageIndex"
+      pagination-style="history"
+      :entities="microblocks"
+      :total-count="keyblockDetails.micro_blocks_count"
+      :limit="limit"
+      @prev-clicked="loadPrevMicroblocks"
+      @next-clicked="loadNextMicroblocks">
+      <keyblock-microblocks-table
+        :microblocks="microblocks"
+        class="keyblock-microblock-panel__table"/>
+      <keyblock-microblocks-table-condensed
+        :microblocks="microblocks"
+        class="keyblock-microblock-panel__table-condensed"/>
+    </paginated-content>
+  </app-panel>
+</template>
+<script setup>
+import { storeToRefs } from 'pinia'
+import { computed } from 'vue'
+import { useKeyblockDetailsStore } from '@/stores/keyblockDetails'
+import PaginatedContent from '@/components/PaginatedContent'
+import KeyblockMicroblocksTable from '@/components/KeyblockMicroblocksTable'
+import KeyblockMicroblocksTableCondensed from '@/components/KeyblockMicroblocksTableCondensed'
+import { isDesktop } from '@/utils/screen'
+
+const { keyblockMicroblocks: microblocks, keyblockDetails } = storeToRefs(useKeyblockDetailsStore())
+const { fetchKeyblockMicroblocks } = useKeyblockDetailsStore()
+const route = useRoute()
+const limit = computed(() => process.client && isDesktop() ? 10 : 3)
+fetchKeyblockMicroblocks({
```

Comment:
I think this should be fetched on the client side only. Right now it's fetched also on server-side but the SSR doesn't wait for the response so you don't see any warnings

Reply:
Thanks, fixed